File: 백준/Bronze/11005. 진법 변환 2/진법 변환 2.py
import sys
input = sys.stdin.readline

N, B = map(int, input().split())
arr = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ'
x = []
while N >= B:
    x.append(N % B)
    N = N // B

x.append(N)
x.reverse()
answer = ''
for i in x:
    answer += arr[i]
print(answer)

File: 백준/Bronze/11005. 진법 변환 2/진법 변환 2.js
let fs = require('fs')
let input = fs.readFileSync("/dev/stdin").toString().trim().split(" ")
// let input = fs.readFileSync("./EX.txt").toString().trim().split(" ")

let N = parseInt(input[0])
const B = parseInt(input[1]) 
const arr = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ'

const x = []

while (N >= B) {
    x.push(N % B)
    N = parseInt(N / B)
}

x.push(N)
x.reverse()

let answer = ''
x.forEach(x => {answer += arr[x]})
console.log(answer)